frontend/src/views/auth/LoginView/LoginValidation.js

```diff
@@ -3,7 +3,11 @@ export default function LoginValidation(values) {
 
   if (!values.email) {
     errors.email = 'Email required';
-  } else if (!/\S+@\S+\.\S+/.test(values.email)) {
+  } else if (
+    !/^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/.test(
+      values.email,
+    )
+  ) {
     errors.email = 'Email address is invalid';
```

Comment:
tutaj value również należy zresetować

Reply:
ojej nie piszmy formularzy i walidacji samemu z palca to samobójstwo xd. polecałbym użyć formika i yupa